internal/services/databricks/databricks_access_connector_resource_test.go

```diff
@@ -45,18 +63,55 @@ func TestAccDatabricksAccessConnector_complete(t *testing.T) {
 	})
 }
 
-func TestAccDatabricksAccessConnector_requiresImport(t *testing.T) {
+func TestAccDatabricksAccessConnector_identityComplete(t *testing.T) {
 	data := acceptance.BuildTestData(t, "azurerm_databricks_access_connector", "test")
 	r := DatabricksAccessConnectorResource{}
-
 	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.complete(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+		{
+			Config: r.identityUserAssigned(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
 		{
 			Config: r.basic(data),
 			Check: acceptance.ComposeTestCheckFunc(
 				check.That(data.ResourceName).ExistsInAzure(r),
 			),
 		},
-		data.RequiresImportErrorStep(r.requiresImport),
+		data.ImportStep(),
+	})
+}
+
+func TestAccDatabricksAccessConnector_identityMissingIdentityIDsError(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_databricks_access_connector", "test")
+	r := DatabricksAccessConnectorResource{}
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config:      r.identityUserAssignedMissingIdentityIDs(data),
+			Check:       acceptance.ComposeTestCheckFunc(),
+			ExpectError: regexp.MustCompile(`must be specified when`),
+		},
+	})
+}
+
+func TestAccDatabricksAccessConnector_identityUserAssignedTooManyError(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_databricks_access_connector", "test")
+	r := DatabricksAccessConnectorResource{}
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config:      r.identityUserAssignedTooManyError(data),
+			Check:       acceptance.ComposeTestCheckFunc(),
+			ExpectError: regexp.MustCompile(`Too many list items`),
+		},
 	})
```

Comment:
fwiw this logic should be handled by the `identity` package, so we should be able to remove these?